avaframe/com1DFAPy/com1DFA.py

```diff
@@ -1707,6 +1715,58 @@ def readPartFromPickle(inDir, flagAvaDir=False):
     return Particles, TimeStepInfo
 
 
+def savePartToCsv(particleProperties, dictList, outDir):
+    """ Save each particle dictionary from a list to a csv file; works also for one dictionary instead of list
+
+        Parameters
+        ---------
+        dictList: list or dict
+            list of dictionaries or single dictionary
+    """
+
+    # set output directory
+    outDir = os.path.join(outDir, 'particlesCSV')
+    fU.makeADir(outDir)
+
+    # read particle properties to be saved
+    particleProperties = particleProperties.split('|')
+
+    # write particles locations and properties to csv file
+    nParticles = len(dictList)
+    count = 0
+    for m in range(nParticles):
+        particles = dictList[count]
+        x = particles['x'] + particles['xllcenter']
+        y = particles['y'] + particles['yllcenter']
+        z = particles['z']
+        simName = particles['simName']
+        partPropDict = {}
+
+        for partProp in particleProperties:
+            if partProp == 'velocityMagnitude':
+                ux = particles['ux']
+                uy = particles['uy']
```

Comment:
do we save no time information in the cvs file?

Reply:
for this reason the csv files have the extension

Reply:
this is done by the csv.0, csv.1, ... I also added an entry in the csv file for the exact time step info